global.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sdk": {
+    "version": "5.0.100-rc.2.20479.15",
+    "allowPrerelease": true,
+    "rollForward": "major"
```

Comment:
Question: we did this for VS experience right?

Reply:
We did this so that you can use a newer SDK when interacting with the repository.